ui/src/react-admin/modules/content-page/components/blocks/BlockEventbrite/BlockEventbrite.tsx

```diff
@@ -38,6 +41,8 @@ export const BlockEventbrite: FunctionComponent<BlockEventbriteProps> = ({
 				});
 			};
 		}
+
+		return () => script.remove();
```

Comment:
Tech used; https://react.dev/reference/react/useEffect#my-cleanup-logic-runs-even-though-my-component-didnt-unmount

> function runs not only during unmount, but before every re-render with changed dependencies

The above shouldn't be an issue here because `elementId` & `eventId` don't change before component unmount